integration_testing/features/copy-multiple-selections.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Copy multiple selections

```

Comment:
Unify this scenario with the previous one into a single `copy-to-clipboard` .feature file with 2 scenarios.